### ISSUE TITLE
security_* metrics - text format parsing error if user tag value has '\' character

### ISSUE THREAD

**Issue by vavdoshka** [NetApp/harvest#1128]
**Describe the bug**

`telegraf` agent is failing to scrape the metric with this error:

```
Jul 01 13:59:00 fr3-inf-wfc-s02 telegraf[2795]: 2022-07-01T13:59:00Z E! [inputs.prometheus] Error in plugin: error reading metrics for http://localhost:12990/metrics: reading text format failed: text format parsing error in line 747: invalid escape sequence '\u'
```

the metric sample is:
```
security_account_labels{datacenter="REDACTED",cluster="REDACTED",user_name="domain\user",applications="http",svm="REDACTED",methods="domain",role_name="readonly",hash_algorithm="",locked=""} 1.0
```

I'm not sure if the problem is specific to telegraf but from what I understood telegraf uses a standard Prometheus parse from where the error is being propagated I guess - https://github.com/influxdata/telegraf/issues/7423

**Environment**

```yaml
Exporters:
  prometheus:
    exporter: Prometheus
    local_http_addr: 0.0.0.0
    port: 12990

Defaults:
  collectors:
    - Zapi
    - ZapiPerf
```

- Harvest version: `harvest version 22.05.0-1`
- Command line arguments used: `harvest  29927  0.2  1.1 977852 45472 ?        Sl   Jun30   2:48 bin/poller --poller default --loglevel 2 --promPort 12990 --config /opt/harvest/harvest.yml --daemon`
- OS: RHEL 7.8
- Install method: yum
- ONTAP Version: 9.7

**To Reproduce**
run harvest with all metrics exposed with collectors `Zapi` and `ZapiPerf`

**Expected behavior**
There is no parsing error 

**Actual behavior**
Parsing error

**Possible solution, workaround, fix**
to filter out the `security_account*` or revert to the previous harvest version, the one which works - `harvest version 21.08.0-6 ` (does not produce `security_account_*` metrics

**Comment by cgrinds** [comment]
thanks for the issue @vavdoshka i think I see the issue.

As a stop gap while we work on a fix, instead of downgrading, you can disable the `  SecurityAccount` object in your `conf/zapi/default.yaml`

**Comment by cgrinds** [comment]
## Notes
On `10.193.48.11`

```
security login create -vserver umeng-aff300-05-06 -user-or-group-name foo\user -application ontapi -authentication-method nsswitch
```

Start Harvest
```
bin/poller --poller u2 --promPort 13002 --objects SecurityAccount
curl -s 'http://127.0.0.1:13002/metrics' | rg foo
```

Before the fix, this is what the Harvest Prometheus exporter publishes
```
security_account_certificateuser{applications="ontapi",cluster="umeng-aff300-05-06",datacenter="dc-1",methods="nsswitch",remote_switch_ipaddress="",svm="umeng-aff300-05-06",user_name="foo\user"} 0
```

which when verified with `promtool` fails

```
curl -s 'http://127.0.0.1:13002/metrics' | promtool check metrics
error while linting: text format parsing error in line 222: invalid escape sequence '\u'
```

With the fix, there are no errors, and the `label_value` is escaped per Prometheus docs
```
security_account_certificateuser{applications="ontapi",cluster="umeng-aff300-05-06",datacenter="dc-1",methods="nsswitch",remote_switch_ipaddress="",svm="umeng-aff300-05-06",user_name="foo\\user"} 0
```

**Comment by cgrinds** [comment]
hi @vavdoshka thanks again for reporting. The issue has been root caused and fixed in #1129. 

If you want to try these changes out before the next release, you can grab the latest [nightly build](https://github.com/NetApp/harvest/releases/tag/nightly).

**Comment by vavdoshka** [comment]
yep @cgrinds works as expected, thx for the quick fix
